ccsdspy/decode.py

```diff
@@ -1,12 +1,92 @@
 """Internal decoding routines."""
 from __future__ import division
 from collections import namedtuple
+from pickle import bytes_types
 
 import numpy as np
 
+from ccsdspy.constants import (
+    BITS_PER_BYTE,
+    PRIMARY_HEADER_NUM_BYTES,
+)
+
 __author__ = "Daniel da Silva <mail@danieldasilva.org>"
 
 
+def _get_packet_total_bytes(primary_header_bytes):
```

Comment:
Do these functions need to be private? Some of them might be helpful to users just like the utility functions. If they are marked as private then they don't show up in the documentation at all which is a shame since they have doc strings!

Reply:
They are exposed via the utils module without the private prefix in the method name.